.conan/build.py

```diff
@@ -32,7 +32,7 @@ def upload(self):
         ci_manager = CIManager(printer)
         branch = ci_manager.get_branch()
 
-        patterns = ["master$", self.stable_branch_pattern]
+        patterns = ["v?\d+\.\d+\.\d+-.*", self.stable_branch_pattern]
```

Comment:
I'm not a Python expert, but I think this line is missing an `r` before the string.

Reply:
Ops! Yeah, you are right.